i18n/translation-en.json

```diff
@@ -162,7 +162,10 @@
   "option": {
     "notesOrder": {
       "position": "Position",
-      "time": "Time"
+      "time": "Time",
```

Comment:
Can you follow https://github.com/XodoDocs/webviewer/wiki/WebViewer-UI-Localization to add translations for the new text?